Shared/mods/deathmatch/logic/lua/CLuaFunctionParser.h

```diff
@@ -444,7 +444,33 @@ struct CLuaFunctionParserBase
         }
         else if constexpr (std::is_same_v<T, CLuaFunctionRef>)
         {
-            return luaM_toref(L, index++);
+        #ifdef MTA_CLIENT
+            CLuaMain* pLuaMain = g_pClientGame->GetLuaManager()->GetVirtualMachine(L);
+        #else
+            CLuaMain* pLuaMain = g_pGame->GetLuaManager()->GetVirtualMachine(L);
+        #endif
+            const void* pFuncPtr = lua_topointer(L, index);
+
+            if (CRefInfo* pInfo = MapFind(pLuaMain->m_CallbackTable, pFuncPtr))
+            {
+                // Re-use the lua ref we already have to this function
+                pInfo->ulUseCount++;
```

Comment:
Is it decreased?

Reply:
Yeah, in the destructor of CLuaFunctionRef

See https://github.com/multitheftauto/mtasa-blue/blob/a0410c68945b6b3d9225c8c90af2bb6b1cda5eaa/Client/mods/deathmatch/logic/lua/CLuaFunctionRef.cpp#L115